src/main/java/com/truelayer/java/payments/IPaymentsApi.java

```diff
@@ -15,15 +15,16 @@
  * @see <a href="https://docs.truelayer.com/reference/create-payment"><i>Payments</i> API reference</a>
  */
 public interface IPaymentsApi {
```

Comment:
I would have loved to use the `default` visibility here but I would need to restructure a lot.